### DISCUSSION TITLE
如何封装一个输入组件，组件包含数值和下拉框

### DISCUSSION THREAD

**Discussion by lppoi** [alibaba/formily#3787] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/64509267/231969263-94018eb1-1990-4278-8ab8-1341431cce1a.png)
类似这样，类型项联动的 输入项下拉框的options, 输出的值格式是123__ng
求一个思路，目前写了个，但是切换单位时没法同步value

**Comment by yanhao98** [comment]
切换单位的下拉框select的change事件的时候你需要emit("change")
